packages/interactor/src/integrations/cypress.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-namespace */
-/// <reference types="cypress" />
+/// <reference types="../../integrations/cypress-interactors/node_modules/cypress" />
```

Comment:
I looked at the resulting package that was published, and it still has this set and obviously doesn't resolve. We may need to look into a way to handle this or can we remove it?

Do you happen to know @minkimcello?

Reply:
I still haven't had a chance to pull the branch to see what's happening. I'll do that soon. I like to generate a preview package and consume it in a separate repository to confirm that everything is working as intended (just to be extra sure).

But from the short talk today, it seems Charles doesn't have any complaints against installing Cypress. Still, having it in the sample app AND in one of the workspace packages means we'll be installing Cypress twice. But from what I can see, there doesn't seem to be a significant difference in time between the runs in this PR and the other PRs.

Reply:
my tsconfig knowledge is very limited but i don't see any errors showing with either of the reference types (i can use either or and restarting the typescript server each time and i don't see any red lines anywhere) 🤷